drivers/google/google.go

```diff
@@ -106,6 +107,11 @@ func GetCreateFlags() []cli.Flag {
 			Usage:  "GCE Instance Tags (comma-separated)",
 			EnvVar: "GOOGLE_TAGS",
 		},
+		cli.BoolFlag{
```

Comment:
Specifically @dmp42 mentions this because the issue is with this flag: it should be of type `mcnflag.BoolFlag` on latest master